sql-statements.md

```diff
@@ -5,6 +5,8 @@ toc: false
 
 CockroachDB supports the following SQL statements. Click a statement for more details.
 
+{{site.data.alerts.callout_info}}For background on how CockroachDB maps SQL table data and table indexes to key-value storage, see <a href="https://www.cockroachlabs.com/blog/sql-in-cockroachdb-mapping-table-data-to-key-value-storage/">SQL in CockroachDB</a>.{{site.data.alerts.end}}
```

Comment:
This doesn't really fit here. Do we have somewhere in the FAQ where we mention that SQL is built on top of KV? That's the right place. If not, just nix this link.